nautobot/dcim/templates/dcim/interface_edit.html

```diff
@@ -18,6 +18,7 @@
             {% render_field form.lag %}
             {% render_field form.mtu %}
             {% render_field form.vrf %}
+            {% render_field form.virtual_device_contexts %}
```

Comment:
I'm not seeing where this field was added to the form definition - was it in a previous PR, or is it missing?